update!: Data broker removal services (#2708)
privacyguides/privacyguides.org: docs/data-broker-removals.md

---
title: Data Removal Services
icon: material/database-off
description: Our recommended methods for removing your personal information from data brokers and people search sites.
cover: data-broker-removals.webp
---

> 

Reviewer: ```suggestion
<small>Protects against the following threat(s):</small>

- [:material-account-search: Public Exposure](basics/common-threats.md#limiting-public-information){ .pg-green }

```

> "People search sites" operated by data brokers represent an immense privacy risk to the majority of Americans. For many, sensitive personal information such as your address, phone number, email, and age are a simple internet search away. While there is unfortunately no federal regulation in place to protect your data, many of these companies will remove your information from their *public* databases upon request.

Reviewer: ```suggestion
"People search sites" operated by data brokers represent an immense privacy risk to the majority of Americans. For many, sensitive personal information such as your address, phone number, email, and age is a simple internet search away. While there is unfortunately no federal regulation in place to protect your data, many of these companies will remove your information from their *public* databases upon request.
```

</div>

## Manual Opt-Outs <small>Free</small>

The quickest, most effective, and most private way to remove yourself from people search sites is to submit opt-out requests manually to each site. This can *seem* like a daunting task, because there are hundreds of people search sites, but the reality is that the vast majority of these sites are operated by a small handful of companies.

You should search for your information on these 8 sites first, and submit an opt-out request if your information is found. Removing your data from these providers typically removes your data from many smaller sites at the same time.

- Acxiom
- BeenVerified
- InfoTracer
- Intelius
- Radaris
- Spokeo
- TruePeopleSearch
- Whitepages

> Once you have done this, it's best to wait a week or two for the requests to propagate to all their sites. Then, you can start to search and opt-out of any remaining sites you find. It can be a good idea to use tools like [Optery](#optery-free--paid)'s free reports or [Google's *Results about you*](#google-results-about-you-free) tool to help find any data that remains on the internet.

Reviewer: ```suggestion
Once you have done this, it's best to wait a week or two for the requests to propagate to all their sites. Then, you can start to search and opt-out of any remaining sites you find. It can be a good idea to use tools like [Optery](#optery-free-paid)'s free reports or [Google's *Results about you*](#google-results-about-you-free) tool to help find any data that remains on the internet.
```

[:simple-github: Big Ass Data Broker Opt-Out List](https://github.com/yaelwrites/Big-Ass-Data-Broker-Opt-Out-List){ .md-button }

If you don't use an automatic scanner to find results about you, consider setting a reminder to re-do this process every 3, 6, or 12 months depending on your risk level and the amount of personal data you have out there. Unfortunately, it is common for your data to re-appear over time or show up on brand new people search sites even after you opt-out.

## EasyOptOuts <small>Paid</small>

<div class="admonition recommendation" markdown>

![EasyOptOuts logo](assets/img/data-broker-removals/easyoptouts.svg){ align=right }

**EasyOptOuts** is a $20/year service which will search a number of different data broker sites and automatically submit opt-out requests on your behalf. They will perform the first search and removal process immediately, and then re-run the process every 4 months in case your data shows up on new sites over time.

[:octicons-home-16: Homepage](https://easyoptouts.com){ .md-button .md-button--primary }
[:octicons-eye-16:](https://easyoptouts.com/privacy){ .card-link title="Privacy Policy" }

</div>

> Some websites supported by EasyOptOuts are publicly searchable, and in those cases EasyOptOuts will perform a search and only submit an opt-out request if your personal data is already found, to prevent sending your data in an opt-out request to sites that didn't have it already. However, they do support some sites which are not publicly searchable, and in those cases your data will be sent to them in an opt-out request regardless, in case you are in their private databases.

Reviewer: ```suggestion
Some websites supported by EasyOptOuts are publicly searchable. In those cases EasyOptOuts will perform a search and only submit an opt-out request if your personal data is already found, to prevent sending your data in an opt-out request to sites that didn't have it already. However, they do support some sites which are not publicly searchable, and in those cases your data will be sent to them in an opt-out request regardless, in case you are in their private databases.
```
Personal preference, but I prefer splitting this long sentence into two

> Our testing indicates that EasyOptOuts provides the best value out of any data removal service we've tested, with a very affordable price and high effectiveness. We will publish a detailed review of EasyOptOuts on our blog in the near future, and update this page when it is published. [Independent findings from Consumer Reports](https://discuss.privacyguides.net/t/consumer-reports-evaluating-people-search-site-removal-services/19948) also indicate that EasyOptOuts is one of the top performing data removal services.

Reviewer: ```suggestion
Our testing indicates that EasyOptOuts provides the best value out of any data removal service we've tested, with a very affordable price and high effectiveness. We will publish a detailed review of EasyOptOuts on our blog in the near future and update this page when it is published. [Independent findings from Consumer Reports](https://discuss.privacyguides.net/t/consumer-reports-evaluating-people-search-site-removal-services/19948) also indicate that EasyOptOuts is one of the top performing data removal services.
```
I don't think the comma pause is needed here.

[:octicons-home-16: Homepage](https://optery.com){ .md-button .md-button--primary }
[:octicons-eye-16:](https://optery.com/privacy-policy){ .card-link title="Privacy Policy" }
[:octicons-info-16:](https://help.optery.com){ .card-link title=Documentation}

</div>

Optery's free scanning tool is an effective way to discover which data brokers have your information already, and their self-service dashboard allows you to easily submit opt-out requests manually.

We have not finished testing Optery's premium, *automatic* opt-out plans, although initial results appear to be positive. [Independent findings from Consumer Reports](https://discuss.privacyguides.net/t/consumer-reports-evaluating-people-search-site-removal-services/19948) also indicate that Optery is one of the top performing data removal services. It necessarily takes at least a few months for us to evaluate a new data removal service, so check back here soon for our own test results. We will also publish a detailed review of Optery's full service on our blog when we have completed testing, and we will update this page accordingly.

## Google *Results About You* <small>Free</small>

> This method will require you to submit your personal information to Google, for them to periodically monitor their search results for. Google claims to not use the information provided to this tool to "personalize your experiences" across other Google products.

Reviewer: ```suggestion
This method will require you to submit your personal information to Google for them to periodically monitor their search results for. Google claims to not use the information provided to this tool to "personalize your experiences" across other Google products.
```
Same thing here

<div class="admonition recommendation" markdown>

![Google logo](assets/img/data-broker-removals/google.svg){ align=right }

> **Results about you** is a free tool which helps you discover whether your personal contact information, including your home address, phone number, and email address, appear in Google search results. If any personal information is found, you can request its removal.

Reviewer: ```suggestion
**Results about you** is a free tool which helps you discover whether your personal contact information, including your home address, phone number, and email address, appears in Google search results. If any personal information is found, you can request its removal.
```

[:octicons-globe-16: Open Web Tool](https://myactivity.google.com/results-about-you){ .md-button .md-button--primary }
[:octicons-info-16:](https://support.google.com/websearch/answer/12719076){ .card-link title=Documentation}

</div>

In many cases, a Google search is the first place a potential stalker or abuser would look to find your personal information, which could make using it a worthwhile trade-off. However, this tool does not remove your information from the discovered websites themselves, only their listings on Google. You should still consider manually opting out from the results which are discovered, or using another service which automatically opts you out from those sites directly.

You can add up to 3 addresses, 3 phone numbers, and 3 email addresses to your Google account to monitor for. The service is only available in select markets (initially the US and UK) to users over 18.

> When results are found, they will be available for review in this web tool. You can also optionally receive an email notification letting you know when new results are found, delivered to the account's Gmail mailbox. You will then be able to click **Request to remove** on each discovered listing, and Google will review the request.

Reviewer: ```suggestion
When results are found, they will be available for review in this web tool. You can also optionally receive an email notification delivered to the account's Gmail address that lets you know when new results are found. You will then be able to click **Request to remove** on each discovered listing, and Google will review the request.
```

> In our testing, this tool worked to reliably remove people search sites from Google search results, but was not effective against websites that showed *corporate* filing information, even if you used your personal address to register a company, nor social media profiles.

Reviewer: ```suggestion
In our testing, this tool worked to reliably remove people search sites from Google search results, but was not effective against websites that showed *corporate* filing information, even if you used your personal address to register a company, nor was it effective against social media profiles.
```

<div class="admonition recommendation" markdown>

**Results about you** is a free tool which helps you discover whether your personal contact information, including your home address, phone number, and email address, appear in Google search results. If any personal information is found, you can request its removal.

[:octicons-globe-16: Open Web Tool](https://myactivity.google.com/results-about-you){ .md-button .md-button--primary }
[:octicons-info-16:](https://support.google.com/websearch/answer/12719076){ .card-link title=Documentation}

</div>

In many cases, a Google search is the first place a potential stalker or abuser would look to find your personal information, which could make using it a worthwhile trade-off. However, this tool does not remove your information from the discovered websites themselves, only their listings on Google. You should still consider manually opting out from the results which are discovered, or using another service which automatically opts you out from those sites directly.

> You can add up to 3 addresses, 3 phone numbers, and 3 email addresses to your Google account to monitor for. The service is only available in select markets (initially the US and UK) to users over 18.

Author: Did you find a list of where this is supported? I was only aware of the two countries mentioned here.
